realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -230,6 +232,25 @@ private static synchronized void removeSession(SyncConfiguration syncConfigurati
         }
     }
 
+    /**
+     * Retruns the all valid sessions belonging to the user.
```

Comment:
Preferer `active` to `valid` since the ObjectStore have uses these 3 states https://github.com/realm/realm-object-store/blob/f462534d94f09a1c6ccb8e161a4eae61a98ebd17/src/sync/sync_session.hpp#L57-L62